.circleci/config.yml

```diff
@@ -155,14 +155,14 @@ commands:
       - run:
           name: Install asdf
           command: |
+            git config advice.detachedHead false
             git config advice.detachedHead false
             if [ ! -d $HOME/.asdf ]; then
               git clone https://github.com/asdf-vm/asdf.git $HOME/.asdf --branch << parameters.asdf_version >>
               cd $HOME/.asdf
             else
               cd $HOME/.asdf
               git fetch origin
-              git fetch --tags
```

Comment:
Not sure that you actually need to `fetch` here – `git pull` below should update the remote anyway